react/base/package.json

```diff
@@ -1,23 +1,23 @@
 {
-  "name": "base",
+  "name": "ionic-app-base",
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@ionic/react": "0.0.5",
-    "react": "~16.8.6",
-    "react-dom": "~16.8.6",
-    "react-router": "~5.0.0",
-    "react-router-dom": "~5.0.0",
-    "react-scripts": "~2.1.8",
-    "typescript": "~3.4.3"
-  },
-  "devDependencies": {
```

Comment:
Curious to know why things like the `@types/*` modules were moved to `dependencies` from `devDependencies`.

Reply:
Create-react-app puts the types in dependencies as well, so it was updated to better match CRA since it's used to create the app.